clients/drcachesim/common/options.cpp

```diff
@@ -504,6 +504,13 @@ droption_t<bytesize_t>
                  "of being simulated.  This skipping may be slow for large skip values; "
                  "consider -skip_instrs for a faster method of skipping.");
 
+droption_t<bytesize_t> op_L0_warmup_refs(
+    DROPTION_SCOPE_CLIENT, "L0_warmup_refs", 0,
+    "Number of memory references for warmup trace",
+    "Specifies the number of memory references included in the warm up trace."
+    "The warmup references come after the fast forward count and before the "
```

Comment:
By "fast forward" this means delayed tracing like -trace_after_instrs?  Maybe best to reword and name that option?

Reply:
Updated text to use trace_after_instrs